src/client/sandbox/cookie/index.js

```diff
@@ -64,7 +68,7 @@ export default class CookieSandbox extends SandboxBase {
         if (!documentCookieIsEmpty)
             nativeMethods.documentCookieSetter.call(this.document, `${clientCookie};expires=${MIN_DATE_VALUE}`);
 
-        return documentCookieIsEmpty;
+        return !documentCookieIsEmpty;
```

Comment:
Wow. Your function worked conversely but tests were passed. May be we should add tests for this?